app/routers/weight.py

```diff
@@ -0,0 +1,67 @@
+from typing import Union
+
+from fastapi import APIRouter, Depends, Request
+from sqlalchemy.orm import Session
+from starlette.responses import RedirectResponse
+
+from app.database.models import User
+from app.dependencies import get_db
+from app.dependencies import templates
+
+
+router = APIRouter(tags=["weight"],)
+
+
+@router.get("/weight")
```

Comment:
you can use prefix prefix="/weight" instead

Reply:
I would do so if I had a nested routes